src/main/java/org/spongepowered/common/mixin/api/mcp/entity/EntityMixin_API.java

```diff
@@ -656,4 +648,29 @@ public EntityArchetype createArchetype() {
         manipulators.add(new SpongeGravityData(!this.hasNoGravity()));
     }
 
+    @Override
+    public Set<Value.Immutable<?>> getValues() {
+        return this.api$getVanillaValues();
+    }
+
+    protected Set<Value.Immutable<?>> api$getVanillaValues() {
+        final Set<Value.Immutable<?>> vanillaValues = new HashSet<>();
+
+        vanillaValues.add(this.displayName().asImmutable());
+        vanillaValues.add(this.passengers().asImmutable());
+        vanillaValues.add(this.onGround().asImmutable());
+        vanillaValues.add(this.velocity().asImmutable());
+        vanillaValues.add(this.gravity().asImmutable());
+
+        if (this.shadow$getLowestRidingEntity() != null) {
+            this.baseVehicle().map(Value::asImmutable).ifPresent(vanillaValues::add);
+        }
+
+        // TODO How to get the creator and notifier?
```

Comment:
Using this api method is fine. It's an implementation toggle whether the `PhaseTracker` will be enabled to track `Owner/Notifier` information (see `ChunkMixin_Tracker` etc.)